lib/github_changelog_generator/parser.rb

```diff
@@ -61,7 +61,7 @@ def self.setup_parser(options)
         opts.on("-b", "--base [NAME]", "Optional base file to append generated changes to.") do |last|
           options[:base] = last
         end
-        opts.on("--bugs-label [LABEL]", "Setup custom label for bug-fixes section. Default is \"**Fixed bugs:**""") do |v|
+        opts.on("--bugs-label [LABEL]", "Setup custom label for bug-fixes section. Default is \"**Fixed bugs:**" "") do |v|
```

Comment:
Oh, right. Long time ago we decide to go with double quotes only, to reduce mess. Anyway there is missed escape character. Thanks. Fixed! Anything else? 😉